src/components/appNavigation/stackNavigation.js

```diff
@@ -29,14 +34,71 @@ const getComponent = (Component, props) => {
   return Component
 }
 
+const getMenuItems = ({ API, sidemenuActions, navigation }) => {
```

Comment:
Is this duplicated from SideMenu? If not this definition should be there right?

Reply:
You are right...Done